tests/specs/parse-tsconfig/extends/extends.spec.ts

```diff
@@ -81,6 +83,31 @@ export default testSuite(({ describe }) => {
 				await fixture.rm();
 			});
 
+			test('absolute path', async () => {
+				const tmpPath = fs.realpathSync(os.tmpdir());
+				const absolutePath = path.join(tmpPath, 'dep/tsconfig.json');
+				const fixture = await createFixture({
+					[absolutePath]: tsconfigJson({
+						compilerOptions: {
+							strict: true,
+							jsx: 'react',
+						},
+					}),
+					'tsconfig.json': tsconfigJson({
+						extends: absolutePath,
+					}),
+					'file.ts': '',
+				});
```

Comment:
You can do `fixture.writeFile()` to create the `tsconfig.json` file that references `fixture.path` for the absolute path.

Reply:
Good idea, fixed